eureka-webapp/src/main/webapp/assets/js/eureka.registration.js

```diff
@@ -137,10 +137,7 @@ window.eureka.registration = new function () {
 				},
 				error: function (xhr, err) {
 					$('#passwordChangeFailure').show();
-					$('#passwordErrorMessage').text(xhr.responseText);
-					$("#passwordErrorMessage:contains('login')").html(function(_, html) {
-                                            return html.replace(/(login)/g, '<a href="/login">$1</a>');
-					});                                        
+					$('#passwordErrorMessage').html("Account already exists! You may go ahead and <a href='protected/login'>login</a>.");                                       
```

Comment:
You need to check the status code and only print your error message if CONFLICT was detected. Otherwise, you need to show a different message or messages.